pandas/core/reshape/pivot.py

```diff
@@ -467,20 +473,20 @@ def crosstab(index, columns, values=None, rownames=None, colnames=None,
 
     >>> foo = pd.Categorical(['a', 'b'], categories=['a', 'b', 'c'])
     >>> bar = pd.Categorical(['d', 'e'], categories=['d', 'e', 'f'])
-    >>> crosstab(foo, bar)  # 'c' and 'f' are not represented in the data,
+    >>> pd.crosstab(foo, bar)  # 'c' and 'f' are not represented in the data,
+    ... # doctest: +SKIP
```

Comment:
What is the point of moving these?

Reply:
Here was the issue, the validator "expects" to see comments in the output (moving the doctest command fixes the problem):
```
Failed example:
    pd.crosstab(foo, bar)  # 'c' and 'f' are not represented in the data,
Expected:
                            # and will not be shown in the output because
                            # dropna is True by default. Set 'dropna=False'
                            # to preserve categories with no data
    ... # doctest: +SKIP
    col_0  d  e
    row_0
    a      1  0
    b      0  1
Got:
    col_0  d  e
    row_0      
    a      1  0
    b      0  1
```

Reply:
How about just making the comments plain text in the examples then?

Reply:
Good idea.  I also noticed that the doctest comments are exposed in the documentation:

https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.crosstab.html

I removed them for `crosstab` and the docstring still validates, so maybe worth trying to do this elsewhere as well?